about.html

```diff
@@ -0,0 +1,96 @@
+    <link rel="stylesheet" href="index.css">
+    <link rel="stylesheet" href="about.css">
+</head>
+<body>
+    <header>
+        <div class="header1">
+            <ul class="header-links">
+                <li><img src="assets/images/facebook-icon.svg" alt=""></li>
+                <li><img src="assets/images/twitterIcon.png" alt=""></li>
+                <li>English</li>
+                <li>my page</li>
+                <li>log in</li>
+            </ul>
+        </div>
+        <nav>
+            <div class="logo-container"><a href="index.html"><img src="assets/images/logo.png" id="logo" alt="logo image"></a></div>
+            <img src="assets/images/menu-icon.svg" id="menu-icon" alt="hamburger menu icon" class="icon">
+            <ul class="nav-links" id="nav-links">
+              <img src="assets/images/menu-close-Icon.png" id="menu-close-icon" class="icon" alt="close menu icon">
+                <li><a href="index.html">Home</a></li>
+                <!-- <div class="divider"></div> -->
+                <li><a href="">program</a></li>
+                <!-- <div class="divider"></div> -->
+                <li><a href="#">Join</a></li>
+                <!-- <div class="divider"></div> -->
+                <li><a href="#">Sponsor</a></li>
+                <!-- <div class="divider"></div> -->
+                <li><a href="#">News</a></li>
+                <!-- <div class="divider"></div> -->
+                <li><a href="#">Campaign</a></li>
+                <!-- <div class="divider"></div> -->
+            </ul>
+        </nav>
+      </header>
+      <main>
+        <section class="one">
+          <span class="sub-title">Hello! Music Producers!</span>
+          <h1 class="title" id="title">Tritone Music Producers conference 2023</h1>
+          <small class="description">The conference is an international, cosmopolitan event that birngs together all manner of artists, tech lovers, music lovers, but mostly music producers, music theorists, and musicians every year.</br> This year, we hope to bring you an experience that will blow your minds (in the best way), and elevate your appreciation of music and the music production process.</br>We invite you to join us if you find yourself within the bracket of different types of individuals mentioned above, or if your imagination has been tickled by the stories or just by visiting this site!</small>
+          <small class="contact-us">Please contact us via Email for any further questions about TMC production conference</small>
+          <a class="contact-link" href="#">Tritonemusic.co@enquiries.org</a>
+        </section>
+        <section class="two">
+            <h3>Tritone Music Co Producers Conference Logo</h2>
+            <div class="divider"></div>
+            <p>The logo of the tritone music co (TMC) is a beacon that stands for great music that measure up to great standards in the industry. It is a beacon of quality that does not compromise beauty!</p>
```

Comment:
- [ ] Kindly recheck the styling for this section. The changes you introduced made the h3 seem to be overlapping the sections that are below it. Kindly fix this.

**This is how I am seeing it now:**

![image](https://user-images.githubusercontent.com/86318284/212384613-9609fda6-b500-47b9-8b2b-e0ca0c9d38b8.png)

**This is the desired design:**

![image](https://user-images.githubusercontent.com/86318284/212384853-52db2108-9115-4040-b870-78015c1a304e.png)

Reply:
fixed